security/badware/index.html

```diff
@@ -3,27 +3,30 @@
 
 <head>
```

Comment:
The page looks like this after changes:

![render](https://github.com/user-attachments/assets/b2a40918-b85f-4fb5-9fbb-978bd2b46caa)